Support/Sources/Delivery/Models/UIView+Data.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  UIView+Data.swift
+//  
+//
+//  Created by Martin Dutra on 28/3/22.
+//
+
+import Foundation
+import UIKit
+
+extension UIView {
```

Comment:
Just a side note, it would be nice to have some kind of shared package in the future that would contain general purpose extensions like this.